src/components/Operations/LaunchMotion/LaunchMotion.js

```diff
@@ -0,0 +1,30 @@
+import React, { useRef, useState, useEffect } from 'react'
+import styles from './launch--motion.module.scss'
+
+const LaunchMotion = (props) => {
+  const iconRef = useRef(null)
+  const [isAnimating, setIsAnimating] = useState(props.isAnimating)
+
+  let iconStyles = {
+    width: props.size,
+    height: props.size
+  }
+
+useEffect(() => {
+    setIsAnimating(props.isAnimating)
+  }, [props.isAnimating])
+
+  const shouldAnimate = isAnimating ? styles.isAnimating : ''
+
+  return (
+    <div ref={iconRef} className={`${shouldAnimate}`}>
+      <svg style={iconStyles} viewBox='0 0 32 32' className={styles.LaunchMotion}>
+        <title>Launch</title>
+        <path  d="M26,28H6a2.0027,2.0027,0,0,1-2-2V6A2.0027,2.0027,0,0,1,6,4H16V6H6V26H26V16h2V26A2.0027,2.0027,0,0,1,26,28Z"/>
```

Comment:
```suggestion
        <path  d="M24.6,6L16,14.6l1.4,1.4L26,7.4L24.6,6z M24,30H4c-1.1,0-2-0.9-2-2V8c0-1.1,0.9-2,2-2h10v2H4v20h20V18h2v10
		C26,29.1,25.1,30,24,30z"/>
```

Reply:
Jumplink looks good to go! Great job! The original launch file was a little cramped so I edited it to allow enough room for the arrow to go forward. I know we originally talked about having a pullback before going forward, but I think it makes more sense in terms of consistency to go with the arrow just moving forward.  Let me know if you are OK with these little tweaks for launch. Overall, they look great!